pandas/io/formats/html.py

```diff
@@ -201,26 +201,6 @@ def _write_header(self, indent):
             # write nothing
             return indent
 
-        def _column_header():
-            if self.fmt.index:
-                row = [''] * (self.frame.index.nlevels - 1)
-            else:
-                row = []
-
-            if isinstance(self.columns, ABCMultiIndex):
-                if self.fmt.has_column_names and self.fmt.index:
-                    row.append(single_column_table(self.columns.names))
-                else:
-                    row.append('')
-                style = "text-align: {just};".format(just=self.fmt.justify)
```

Comment:
Relatively nuanced but in the case of a MultiIndex isn't this formatting the very first column row different from the rest? If so is that accounted for in the refactor?

Reply:
This `if` clause is unreachable since `_column_header()` was only called from the `else` clause of another `if isinstance(self.columns, ABCMultiIndex):` so the styling is not accounted for in this refactor.